procedures/CodeBrowser_utils.ipf

```diff
@@ -83,10 +83,19 @@ Function GetScreenDimensions(rect)
 End
 
 // Outputs a debug message prefixed with the calling function of debugPrint
-Function debugPrint(msg)
+//
+// @param loglevel   debug level
+// 					 0: No Error
```

Comment:
@ukos-git Wrong indentation.

Reply:
@ukos-git I never found a syntax checker for doxygen comments.

Reply:
i added a quick Doxyfile to the commit but it is not complaining. It is not even finding the functions correctly. Did I miss something.

Reply:
@ukos-git Is the awk script present and in the correct location?

Reply:
https://pastebin.com/kf1Gmgbz

script is present and executed but only struct is recognized. No functions.

Reply:
@ukos-git Fixed and pushed. You gotta use the example from the doxygen-filter-ipf repo, this has EXTRACT_ALL set to yes.